### ISSUE TITLE
Missing Visual Focus on Cart Page

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4319]
![Screen Shot 2019-08-29 at 2 49 01 PM](https://user-images.githubusercontent.com/15113219/63968453-bc3ef780-ca6d-11e9-907e-e378a714203a.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔
- Firefox: ✔ (note: focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome & Firefox: Only one styling issue with the border when the focus is on the - sign.
<img width="1133" alt="Screen Shot 2019-10-28 at 2 10 29 PM" src="https://user-images.githubusercontent.com/45409515/67705411-54356080-f98d-11e9-9099-3040dc21d0d3.png">


On safari, they are missing for these:
- and + signs
remove button
Proceed to checkout button

**Comment by developpeurweb** [comment]
Focus is now visible in Safari, although the cropped focus on the minus button persists, it seems like `z-index` issue, so i'll try fixing it from that perspective.

**Comment by developpeurweb** [comment]
**For QA:**

- For now the visible focus outline only works when theme `calydon` is enabled.
- For Safari testing please refer to the settings detailed in ticket GH-5041.

**Comment by developpeurweb** [comment]
![Screen Shot 2020-01-29 at 6 30 50 PM](https://user-images.githubusercontent.com/15113219/73406883-8d9ba600-42c5-11ea-989b-72823adecd31.png)

**Comment by KateChuen** [comment]
Tested on dev3.
chrome ✅
firefox ✅
safari ✅
